vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gatt_server.c

```diff
@@ -509,63 +403,109 @@ static int prvGATTCharAccessCb( uint16_t conn_handle,
     return rc;
 }
 
+bool setNewHandle( const ble_uuid_t * uuid,
```

Comment:
looks like this is a private function.  Set function type to static and prefix with prv.

Reply:
Thank you! Fixed!